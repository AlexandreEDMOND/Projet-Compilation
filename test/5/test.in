int main()
{
  float a = 8.00;
  float b = 1856.75982;
  float c = -86.34;

  print(a);
  print(b);
  print(c);
  
  return 0;
}