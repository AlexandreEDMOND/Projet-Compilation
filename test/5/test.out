8.00
1856.75982
